SymHop/src/SymHop.cpp

```diff
@@ -2624,7 +2628,7 @@ void Expression::expand(const ExpressionSimplificationT simplifications)
 //Expand power functions from e.g. "pow(x,3)" to "x*x*x". This will improve performance in generated code.
 void Expression::expandPowers()
 {
-    if(this->isPower() && this->getPower()->isNumericalSymbol()) {
+    if(this->isPower() && this->getPower()->isInteger()) {
         int nFactors = int(mpPower->toDouble());
```

Comment:
This probably works `int(mpPower->toDouble());` but i would feel better if it was `int(mpPower->toDouble()+0.5);` but I am not sure if it matters. At least for small doubles they probably convert cleanly to int.

Reply:
Good point. It's not very common to use very large integer exponents, but better be safe.